App/SB/components/FeedItem/index.tsx

```diff
@@ -48,6 +48,9 @@ const FeedItem = (props: FeedItemProps) => {
     </View>
   )
 
+  const feedText = payload.feed && payload.feed !== '' ? payload.feed : undefined
```

Comment:
I'm not sure what happens if you render `undefined` in a `Text` component. Might get the same crash?

Reply:
the ticket i shared says, no

Reply:
Think one of us is reading that wrong. To me, it's talking about that in the the condition `{this.state.error && ` will only work as intended when `error` is `undefined` and not `''`. They are avoiding every rendering an empty string by putting some static text in there with a possibly `undefined` `this.state.error`:
```
<Text>
    Error message: {this.state.error}
  </Text>
```

Reply:
So there is always a string with length in the `Text` component